source/code-snippets/crud/bulk.js

```diff
@@ -16,19 +16,19 @@ const uri = '<connection string>'; // Add your MongoDB connection string here
 
         // begin-sample-data
         const docs = [
-            { title: "Inception", year: 2010, rated: "PG-13", released: "2010-07-16" },
-            { title: "Interstellar", year: 2014, rated: "PG-13", released: "2014-11-07" },
-            { title: "The Dark Knight", year: 2008, rated: "PG-13", released: "2008-07-18" },
-            { title: "Tenet", year: 2020, rated: "PG-13", released: "2020-09-03" }
+            {title: "Inception", year: 2010, rated: "PG-13", released: "2010-07-16"},
```

Comment:
[s] I don't think these spacings needed to removed, sorry if my last comment was confusing! I meant the indentation spacing:
```suggestion
          { title: "Inception", year: 2010, rated: "PG-13", released: "2010-07-16" },
```